File: Python/Ex16.py
l1= float(input("Digite o primeiro lado do triangulo: "))
l2= float(input("Digite o segundo lado do triangulo: "))
l3= float(input("Digite o terceiro lado do triangulo: "))

if ((l1 + l2) > l3) and ((l1 + l3) > l2) and ((l2 + l3) > l1):
    if (l1 == l2) and (l1 == l3) :
        print("Triangulo Equilatero")
    elif (l1 != l2) and (l1 != l3):
        print("Triangulo Escaleno")
    else: 
        print("Triangulo Isosceles")  

else: 
    print("Nao eh um Triangulo")          

File: C/Ex16.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[]){
    float l1, l2, l3;

    printf("Digite o primeiro lado do triangulo: ");
    scanf("%f", &l1);
    printf("Digite o segundo lado do triangulo: ");
    scanf("%f", &l2);
    printf("Digite o terceiro lado do triangulo: ");
    scanf("%f", &l3);

    if ( ((l1 + l2) > l3) &&
       ((l1 + l3) > l2) &&
       ((l2 + l3) > l1) ){
           if ( (l1 == l2) && (l1 == l3) ){
               printf("Triangulo Equilatero");
           }
           else if( (l1 != l2) && (l1 != l3) ){
               printf("Triangulo Escaleno");
           }
           else{
               printf("Triangulo Isosceles");
           }
       }
       else{
           printf("Nao eh um Triangulo");
       }
 
  return 0;
}

